source/manual/get-started.html.md

```diff
@@ -317,18 +317,21 @@ Note this may happen even if you don't use an rsa ssh private key - it is caused
 
 Follow [the instructions for setting up tools to use the GOV.UK Kubernetes platform](/kubernetes/get-started/set-up-tools/).
 
-## 12. Get a Signon account for integration
+## 12. Get Signon accounts for integration and production
 
 [Signon](/repos/signon.html) is the application used to control access to the GOV.UK Publishing applications.
 
-Ask your tech lead to [create an account for the integration Signon](https://signon.integration.publishing.service.gov.uk/users/invitation/new), at 'Superadmin' level with permission to access the applications that your team are likely to work on.
+Ask your tech lead to give you the following access:
 
-## 13. Get access to the Release app
+- 'Superadmin' level with permission to access the applications that your team are likely to work on [for the integration Signon](https://signon.integration.publishing.service.gov.uk/users/invitation/new).
+- 'Normal' level with access to the 'Release' app only (no permissions should be given for other applications, until [production access](/manual/rules-for-getting-production-access.html) is granted) [for the production Signon](https://signon.publishing.service.gov.uk/users/invitation/new).
```

Comment:
```suggestion
- 'Normal' level with access to the 'Release' app only (no permissions should be given for other applications, until [production access](/manual/rules-for-getting-production-access.html) is granted) [for the production Signon](https://signon.publishing.service.gov.uk/users/invitation/new). Note that this will automatically grant you access to the [staging Signon](https://signon.staging.publishing.service.gov.uk/) environment once the nightly [env sync](/manual/govuk-env-sync.html) has completed.
```

Reply:
Nice idea.  Have updated.